internals/plan/plan.go

```diff
@@ -796,35 +764,32 @@ func CombineLayers(layers ...*Layer) (*Layer, error) {
 			}
 		}
 
-		switch target.Selection {
-		case OptOutSelection, OptInSelection, DisabledSelection, UnsetSelection:
-			// valid, continue
-		default:
+		// Validate service names specified in log target
+		for _, serviceName := range target.Services {
+			if serviceName == "all" || serviceName == "-all" {
+				continue
+			}
+			if _, ok := combined.Services[serviceName]; ok {
+				continue
+			}
+			// Check if this is `-svc`
+			serviceName = strings.TrimPrefix(serviceName, "-")
+			if _, ok := combined.Services[serviceName]; ok {
+				continue
+			}
 			return nil, &FormatError{
-				Message: fmt.Sprintf(`log target %q has invalid selection %q, must be %q, %q or %q`,
-					name, target.Selection, OptOutSelection, OptInSelection, DisabledSelection),
+				Message: fmt.Sprintf(`log target %q specifies unknown service %q`,
+					target.Name, serviceName),
 			}
 		}
 
-		if target.Location == "" && target.Selection != DisabledSelection {
+		if target.Location == "" && len(target.Services) > 0 {
```

Comment:
Why do we only validation `location` if services is set? Shouldn't we always do that regardless?

Reply:
I think so you can disable a target with
```
tgt1:
  services: []
  override: replace
```

We don't really need `location` if the target is not receiving any logs. I'm open to changing this though.

Reply:
Yeah, I realize we don't need it, but it just seems weird for the validation of `location` to depend on whether it's going somewhere right at the moment or not.

Reply:
I guess now you can disable a target by merging this on top
```
tgt1:
  services: [-all]
  override: merge
```
so we can remove this exception to the rule.